kod/object/active/holder/nomoveon/battler/monster/towns/Goad.kod

```diff
@@ -791,12 +792,9 @@ messages:
    "Someone in the room said 'champion'.  Net effect, if there is no "
    "champion, then make the talker the new champion."
    {
-      if piCombat_style = STYLE_NO_FIGHT
-         {
-           %%Only one style availible: Duel.
-           %send(self,@say,#message_rsc=Goad_must_choose_style);
-           %return;    
-           post(self,@ChooseCombat,#style = STYLE_ONE_ON_ONE,#actor=what); 
+      if piCombat_style <> STYLE_ONE_ON_ONE
+         {  
+           Send(self,@ChooseCombat,#style = STYLE_ONE_ON_ONE); 
```

Comment:
This is confusing.  If the style isn't 1-on-1, make it 1-on-1?  Is the "if" needed, or it is safe to simply always set the style to 1-on-1?

Reply:
Yes we don't need an if or even a choose combat message.  I was trying to fix the potential problem without re-writing too much of the machinery.

Its possible that re-writing the machinery is the better path.  The simplest re-write would be removing ChooseCombat and rolling its function into the Trig functions. 

I also don't want to leave this particular game-update-bug in production even though it only comes once after each recreate.